WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/ContactSupportScreen.java

```diff
@@ -67,6 +70,7 @@ public HelpAndSupportScreen goBackAndDeleteUnsentMessageIfNeeded() {
 
     // Assertions:
     public ContactSupportScreen assertContactSupportScreenLoaded() {
+        waitForElementToBeDisplayed(inputTextField);
```

Comment:
I wonder if this will work on FTL? (it does locally)

```suggestion
        waitForElementToBeDisplayed(textInput);
```

Then we won't need the `inputTextField` added above.

Reply:
Good catch, it does work! ([FTL test run](https://console.firebase.google.com/project/api-project-108380595987/testlab/histories/bh.f7bbef01bccfafb8/matrices/5073619353866657112/executions/bs.1e706ca8c19bb56a/test-cases)). 

Change made here: https://github.com/wordpress-mobile/WordPress-Android/pull/17312/commits/a4186d6786403006a20048d30ae26b66e4124e67